plugins/dynamix/Browse.page

```diff
@@ -23,6 +23,7 @@ function sort_by($field, $opt, $show_disk) {
 // read directory contents into 'list' array
   global $docroot,$dir;
   $path = $docroot.preg_replace('/([\'" &()[\]\\\\])/','\\\\$1',$dir).'/*';
+  file_put_contents('/tmp/test',$path);
```

Comment:
Good catch, used it for debugging. Made another pull request with the correction.